open_academy/models/course.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
```

Comment:
This model is not being loaded as it is not present in `open_academy/models/__init__.py`, import it